src/pages/index.js

```diff
@@ -0,0 +1,285 @@
+import "./index.css";
+import {
+  enableValidation,
+  settings,
+  disableButton,
+  resetValidation,
+} from "../scripts/validation.js";
+import Api from "../utils/Api.js";
+import { setButtonText } from "../utils/helpers.js";
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "070c6453-a0bb-46aa-bb30-574e3fcfc6eb",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getAppInfo()
+  .then(([cards, users]) => {
+    cards.forEach((item) => {
+      const cardElement = getCardElement(item);
+      cardsList.prepend(cardElement);
+    });
+
+    profileName.textContent = users.name;
+    profileDescription.textContent = users.about;
+    profileAvatar.src = users.avatar;
+  })
+  .catch(console.error);
+
+// Profile elements
+
+const avatarModalBtn = document.querySelector(".profile__avatar-btn");
+const profileName = document.querySelector(".profile__name");
+const profileDescription = document.querySelector(".profile__description");
+const profileEditButton = document.querySelector(".profile__edit-btn");
+const profileAddButton = document.querySelector(".profile__add-btn");
+const profileAvatar = document.querySelector(".profile__avatar");
+
+// Avatar form elements
+
+const avatarModal = document.querySelector("#avatar-modal");
+const avatarForm = avatarModal.querySelector(".modal__form");
+const avatarModalSaveBtn = avatarModal.querySelector(".modal__submit-btn");
+const avatarModalCloseBtn = avatarModal.querySelector(".modal__close-btn");
+const avatarInput = avatarModal.querySelector("#profile-avatar-input");
+
+// Edit form elements
+const editModal = document.querySelector("#edit-modal");
+const editFormElement = editModal.querySelector(".modal__form");
+const modalCloseButton = editModal.querySelector(".modal__close-btn");
```

Comment:
`modalCloseButton` 


There are 3 modals with different buttons and forms in the project. You need to **name** variables and functions the way all could understand what they are: `profileForm`, `profileCloseButton` and so on